identity/convert.go

```diff
@@ -0,0 +1,59 @@
+// This file is part of the go-meta library.
+//
+// Copyright (C) 2017 JAAK MUSIC LTD
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+// If you have any questions please contact yo@jaak.io
+
+package identity
+
+import (
+	"github.com/ipfs/go-cid"
+	"github.com/meta-network/go-meta"
+)
+
+// Converter converts Identity data in meta object
+// objects.
```

Comment:
Given all the data about an identity is stored in the index (i.e. the `owner`, `id` and `signature`), is there any need to create a META object?